index.html

```diff
@@ -183,5 +183,25 @@ <h3>
                     </ul>
             </section>
     </div>
+    <section id="contact">
+        <h2>Contact me</h2>
+        <p>
+            If you have an application you are intersted in developing, 
+            a feature you that you need built or a project that needs coding. 
+            I'd love to help with it!
+        </p>
+        <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] Currently, your form submission raises an error. Kindly create a form with [formspree service](https://formspree.io/html) and use the unique id generated in your `action` prop.
 
![image](https://user-images.githubusercontent.com/98453979/228296807-a66bef7d-5235-47fe-9ad3-9a8944acef59.png)